services/webnn/webnn_switches.h

```diff
@@ -35,6 +35,9 @@ inline constexpr char kWebNNUseOrt[] = "webnn-use-ort";
 // Usage: --no-sandbox --webnn-ort-dump-model=./OnnxModels
 inline constexpr char kWebNNOrtDumpModel[] = "webnn-ort-dump-model";
 
+// Use DirectML EP of ONNX Runtime
+inline constexpr char kWebNNOrtUseDml[] = "webnn-ort-use-dml";
```

Comment:
A nit: `inline constexpr char kWebNNOrtUseDmlGpu[] = "webnn-ort-use-dml-gpu";`? to make it clear that you will use DML only for GPU device.

Reply:
done.